examples/cd_text_amazon.ipynb

```diff
@@ -0,0 +1,553 @@
+    "        #   test data is used but only out-of-fold predictions taken into account for the drift detection\n",
+    "        #   alternatively we can set train_size to a fraction between 0 and 1 and not apply cross-validation\n",
+    "        # epochs specifies how many epochs the classifier will be trained for each sample or fold\n",
+    "        # preprocess_batch_fn is applied to each batch of instances and translates the text into tokens\n",
+    "        dd = ClassifierDrift(x_ref, model, backend=backend, p_val=.05, preds_type='logits', \n",
+    "                             n_folds=3, epochs=2, preprocess_batch_fn=batch_fn, train_size=None)\n",
+    "        preds_h0 = dd.predict(x_h0)\n",
+    "        preds_h1 = dd.predict(x_h1)\n",
+    "        print_preds(preds_h0, 'H0')\n",
+    "        print_preds(preds_h1, 'H1')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "H0\n",
+      "Drift? No!\n",
+      "p-value: 0.644\n",
+      "\n",
+      "H1\n",
+      "Drift? Yes!\n",
+      "p-value: 0.006\n",
+      "\n",
+      "H0\n",
+      "Drift? No!\n",
+      "p-value: 0.697\n",
+      "\n",
+      "H1\n",
+      "Drift? Yes!\n",
+      "p-value: 0.003\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "make_predictions(model, 'pytorch', ref_size=1000, test_size=1000, n_sample=2, stars_h1=4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### TensorFlow drift detector\n",
+    "\n",
+    "We can do the same using TensorFlow instead of PyTorch as backend. We first define the classifier again and then simply run the detector:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Some layers from the model checkpoint at distilbert-base-uncased were not used when initializing TFDistilBertModel: ['vocab_transform', 'vocab_projector', 'activation_13', 'vocab_layer_norm']\n",
+      "- This IS expected if you are initializing TFDistilBertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
+      "- This IS NOT expected if you are initializing TFDistilBertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n",
+      "All the layers of TFDistilBertModel were initialized from the model checkpoint at distilbert-base-uncased.\n",
+      "If your task is similar to the task the model of the checkpoint was trained on, you can already use TFDistilBertModel for predictions without further training.\n"
```

Comment:
Lots of warnings showing around this part of the notebook.

Reply:
Yes, internal transformers library warnings unfortunately. Tried with warnings in the notebook explicitly switched off but that didn't help.